Classes/BEMPathManager.m

```diff
@@ -16,13 +16,13 @@ - (UIBezierPath *)pathForBox {
     UIBezierPath* path;
     switch (self.boxType) {
         case BEMBoxTypeSquare:
-            path = [UIBezierPath bezierPathWithRoundedRect:CGRectMake(0, 0, self.size, self.size) cornerRadius:self.cornerRadius];
+            path = [UIBezierPath bezierPathWithRoundedRect:CGRectMake(0.0 + (_lineWidth / 2.0), 0.0 + (_lineWidth / 2.0), self.size - _lineWidth, self.size - _lineWidth) cornerRadius:self.cornerRadius];
```

Comment:
Nit: The rest of the project uses `self` to access properties instead of ivars. Would you mind replacing `_lineWidth` with `self.liveWidth` for consistency?